openfisca_core/tools/test_runner.py

```diff
@@ -174,7 +174,7 @@ def _generate_tests_from_directory(tax_benefit_system, path_to_dir, options):
 def _parse_test_file(tax_benefit_system, yaml_path, options):
     with open(yaml_path) as yaml_file:
         try:
-            tests = yaml.load(yaml_file)
+            tests = yaml.safe_load(yaml_file)
```

Comment:
```suggestion
            tests = yaml.load(yaml_file, Loader = Loader)
```

Then in https://github.com/openfisca/openfisca-core/pull/845/files#diff-68d30ead1f6e6a74cfa83304a419aaa8R44 change to:

```python
from yaml import SafeLoader as Loader
```

Otherwise, test loading time will increase dramatically.